src/components/exercises/ExerciseForm.jsx

```diff
@@ -0,0 +1,88 @@
+import React, { useState, useEffect } from 'react';
+import { withRouter } from 'react-router';
+
+import './ExerciseForm.css';
+import { Input, Modal, Radio, Button } from 'antd';
+
+import { ImagePicker, Card } from '../custom';
+
+const { TextArea } = Input;
+
+const ExerciseForm = ({ title, exercise, onChange, onSave, history }) => {
+    const handleInputChange = (inputName) => ({target : {value}}) => {
+        const modifiedExercise = {...exercise};
+        modifiedExercise[inputName] = value;
+        onChange(modifiedExercise);
+    };
+
+    const handleImageChange = (image) => onChange({
+        ...exercise,
+        image
+    });
+
+    const handleRadioChange = (e) => {
+      console.log('radio checked', e.target.value);
+    };
+
+    const { name, description, image } = exercise;
+
+    const exerciseTypes = [
+        {
+            name: 'Théorique',
+            value: 'THEORETICAL'
+        },
+        {
+            name: 'Physique',
+            value: 'PHYSICAL'
+        },
+        {
+            name: 'Taolu',
+            value: 'TAOLU'
+        },
+        {
+            name: 'Combat',
+            value: 'FIGHT'
+        }
+    ];
+
+    return (
+        <div className="ExerciseForm">
+            <Card className="card">
+                <h1>{title}</h1>
+                <h2 className="infos-title">Informations</h2>
+                <div className="exerciseType">
+                  <Radio.Group onChange={handleRadioChange}>
+                    {exerciseTypes.map((type, index) =>
+                      <Radio value={type.value}>{type.name}</Radio>
```

Comment:
or is this a 2-way binding?

Reply:
It's a 2-way binding but this code isn't finished yet. I shouldn't have been in this PR. It was a way to make the code progress but it will be changed in the next PR with the creation of exercises.